client/packages/common/src/ui/components/inputs/TextInput/InputWithLabelRow.tsx

```diff
@@ -15,17 +16,29 @@ export const InputWithLabelRow: FC<InputWithLabelRowProps> = ({
   Input = <BasicTextInput />,
   labelProps,
   labelWidth = '120px',
+  labelRight = false,
   sx,
 }) => {
   const { sx: labelSx, ...labelPropsRest } = labelProps || {};
 
   return (
-    <Box display="flex" alignItems="center" gap={1} sx={sx}>
+    <Box
+      sx={{
+        ...sx,
+        display: 'flex',
+        alignItems: 'center',
+        gap: 1,
+        ...(labelRight
+          ? { gap: 2, flexDirection: 'row-reverse', justifyContent: 'flex-end' }
+          : {}),
```

Comment:
It probably won't make much of a difference, but we'd normally put `...sx` last so that the defined properties can be overridden if needed :)